documentation/modules/overview/con-configuration-points-connect.adoc

```diff
@@ -5,127 +5,257 @@
+To manage connectors with `KafkaConnector` resources, you must specify an annotation in your `KafkaConnect` custom resource.
 
-The `spec` for the KafkaConnector resource specifies the connector class and configuration settings, as well as the maximum number of connector _tasks_ to handle the data.
+.Annotation to enable KafkaConnectors
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaConnectApiVersion}
+kind: KafkaConnect
+metadata:
+  name: my-connect-cluster
+  annotations:
+    strimzi.io/use-connector-resources: "true"
+  # ...
+----
 
-[discrete]
-== Example YAML showing KafkaConnector configuration
+Setting `use-connector-resources` to `true`  enables KafkaConnectors to create, delete and reconfigure connectors.
+
+If enabled in your `KafkaConnect` configuration, you must use the `KafkaConnector` resource to define connectors.
+`KafkaConnector` resources are configured to connect to external systems.
+They are deployed to the same Kubernetes cluster as the Kafka Connect cluster and Kafka cluster interacting with the external data system.
+
+.Kafka components are contained in the same Kubernetes cluster
+image:overview/kafka-concepts-kafka-connector.png[Kafka and Kafka Connect contained in the same cluster]
+
+The configuration specifies how connector instances connect to an external system, including any authentication.
+You also need to say what data to watch within the data source.
+For example, you might provide a database name in the configuration.
+You can also specify where the data should sit in Kafka by specifying a target topic name.
+
+.Example KafkaConnector configuration
 [source,yaml,subs="attributes+"]
 ----
-apiVersion: {KafkaConnectorApiVersion}
+apiVersion: {KafkaConnectApiVersion}
 kind: KafkaConnector
 metadata:
-  name: my-source-connector
+  name: my-source-connector  <1>
   labels:
-    strimzi.io/cluster: my-connect-cluster
+    strimzi.io/cluster: my-connect-cluster <2>
 spec:
-  class: org.apache.kafka.connect.file.FileStreamSourceConnector
-  tasksMax: 2
-  config:
-    file: "/opt/kafka/LICENSE"
-    topic: my-topic
+  class: org.apache.kafka.connect.file.FileStreamSourceConnector <3>
+  tasksMax: 2 <4>
+  config: <5>
+    file: "/opt/kafka/LICENSE" <6>
+    topic: my-topic <7>
     # ...
 ----
+<1> Name of the `KafkaConnector` resource, which is used as the name of the connector. Use any name that is valid for a Kubernetes resource.
+<2> Name of the Kafka Connect cluster to create the connector instance in. Connectors must be deployed to the same namespace as the Kafka Connect cluster they link to.
+<3> Full name or alias of the connector class. This should be present in the image being used by the Kafka Connect cluster.
+<4> Maximum number of Kafka Connect tasks that the connector can create.
+<5> link:{BookURLDeploying}#kafkaconnector-configs[Connector configuration^] as key-value pairs.
+<6> Location of the connector configuration file. The source connector reads from the `/opt/kafka/LICENSE` file.
```

Comment:
Make is a bit clearer that this configuration is specific to the connector class.

```suggestion
<6> In this example we're configuring the FileStreamSourceConnector to read from the `/opt/kafka/LICENSE` file.
```